x-pack/plugin/sql/qa/server/src/main/resources/select.csv-spec

```diff
@@ -241,4 +241,12 @@ Weiyi          |WEIYI
 Vishv          |VISHV          
 Valter         |VALTER         
 Valdiodio      |VALDIODIO
-;
\ No newline at end of file
+;
+
+ambiguousReferenceInWhereClause
+SELECT gender emp_no, emp_no e FROM test_emp WHERE e > 10099;
```

Comment:
You could add the raw column (gender) in the select to further reinforce the point:
`SELECT gender, gender emp_no, ...`
so the result contains both gender and emp_no with the same value.